evm/src/util.rs

```diff
@@ -179,7 +176,6 @@ pub(crate) fn biguint_to_le_limbs(x: BigUint) -> Vec<u128> {
         .collect()
 }
 
-#[cfg(test)]
 pub(crate) fn biguint_to_mem_vec(x: BigUint) -> Vec<U256> {
     biguint_to_le_limbs(x)
         .into_iter()
```

Comment:
I think this would be cleaner (and faster) with no `Vec`s involved. `biguint_to_le_limbs` `collect`s an iterator into a vector, only for the vector to be read as an iterator. So it shouldn't be a vector in the first place!

Reply:
Hm does this change fix it? (Getting rid of the `le_limbs` functions entirely)